src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -588,8 +595,7 @@ describe('sidebar.components.sidebar-content', function() {
     }
 
     beforeEach(function() {
-      // There is a direct-linked annotation
-      fakeSettings.annotations = 'test';
+      store.setDirectLinkedAnnotationsId('test');
```

Comment:
Hmm, I can't remember if multiple annotation IDs are actually supported. If not I'd change this to singular rather than plural.